src/pages/trade/ui/chart.tsx

```diff
@@ -142,31 +144,34 @@ const ChartData = observer(({ candles }: { candles: OhlcData[] }) => {
           },
         },
       });
+
+      // Initialize the series
+      seriesRef.current = chartRef.current.addCandlestickSeries({
+        upColor: theme.color.success.light,
+        downColor: theme.color.destructive.light,
+        borderVisible: false,
+        wickUpColor: theme.color.success.light,
+        wickDownColor: theme.color.destructive.light,
+      });
```

Comment:
The issue before was that candle updates was adding an additional chart and not editing the one in place